src/metro/index.ts

```diff
@@ -44,6 +46,14 @@ for (const key in modules) {
         }
     }
 
+    if (!themeModuleFound && module?.publicModule?.exports?.ThemeColorMap) {
+        // Theme colors are overwritten here
+        themerInit(module.publicModule.exports);
+
+        themeModuleFound = true;
+        continue;
+    }
```

Comment:
should you not error once you've looped over each and theme module still wasn't found?